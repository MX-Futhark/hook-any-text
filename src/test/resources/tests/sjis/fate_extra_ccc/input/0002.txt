90b396e582c982cd92f682c882ad82b582c4939e928582b582bd814200ffffff00000000000100000002000000030000000400000005000000060000000700000008000000090000000a0000000b0000000c0000000d000004000000f014c90800007041000084420000803f0000803f0000000000002000260000002800000080000000ffffffff0002000000010000ffffffff0f000000180000005814c9080814c9084c4fa34800000000000000008e9e8ad482cd82dc82be8f5c95aa82c982a082e981420a93af82b690a7959e8e7082cc90b6936b82bd82bf82aa814198618b43e55c815882c690b396e582f092ca89df82b582c482a282ad814200c90800000000000100000002000000030000000400000005000000060000000700000008000000090000000a0000000b0000000c0000000d0000000e0000000f000000100000001100000012000000130000001400000015000000160000001700000018000000190000001a0000001b0000001c0000001d0000001e0000001f000000200000002100000022000000230000002400000025000004000000bc15c908000070410000fc420000803f0000803f00000000000020000f0000002800000080000000ffffffff0002000000010000ffffffff0f000000180000008015c9084815c908945da34800000000000000002352554253838b815b8360839323525542458f4b8ab52352454e4492ca82e881418ea995aa82e090b396e582c98cfc82a982a4814200000000000000000100000002000000030000000400000005000000060000000700000008000000090000000a0000000b0000000c0000000d0000000e00000b0000000000000000000000000000000000000000001f01000000000000000000000000ffffffff000075430000ee42000000000000604100000000ffffffff008081430000ee42000000000000000000006041ffffffff0000754300000543000000000000604100006041