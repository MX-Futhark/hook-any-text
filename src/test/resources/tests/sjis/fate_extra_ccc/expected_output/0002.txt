["正門には程なくして到着した。",
"時間はまだ十分にある。
同じ制服姿の生徒たちが、和気藹々と正門を通過していく。",
"習慣通り、自分も正門に向かう。"]